src/components/FormElements/OrderPicker.tsx

```diff
@@ -0,0 +1,29 @@
+import { RadioPicker } from "@/components/RadioPicker";
+
+type OrderPickerProps = {
+  id: string;
+  selectedOption: string | undefined;
+  onChange: (optionId: string | undefined, optionValue?: string) => void;
+  labelSuffix?: string | React.ReactNode;
+};
+
+export const OrderPicker = ({
+  id,
+  selectedOption,
+  onChange,
+  labelSuffix,
+}: OrderPickerProps) => {
```

Comment:
we can just do `( )` and remove `{ }` and `return`

Reply:
Good call! I'll update.